src/db/oracledb/addresses-dao.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+
+import { serializeAddresses } from 'serializers/addresses-serializer';
+import { parseQuery } from 'utils/parse-query';
+import { getConnection } from './connection';
+import { contrib } from './contrib/contrib';
+
+/**
+ * Queries data source for raw address data and passes it to the serializer before returning
+ *
+ * @param {string} osuId OSU ID of person to select addresses from
+ * @param {object} query Query parameters passed in with request
+ * @returns {Promise<object>} Serialized address resource
+ */
+const getAddressesById = async (osuId, query) => {
```

Comment:
Not sure about this, but would it be better to call this `getAddressesByOsuId` instead to be more specific?